### DISCUSSION TITLE
Should we only use `defaultTheme` and `themes` key?

### DISCUSSION THREAD

**Discussion by FranciscoKloganB** [RyanClementsHax/tailwindcss-themer#35] [Q&A] [answered]
Should we only use `defaultTheme` and `themes` key in `tailwind.config.js` or can we keep on using `theme.extend` as a default default?

**Comment by RyanClementsHax** [comment, marked as answer]
Hiya! Thanks for your question.

I'm not sure what you mean by a "default default", but you can reference the documentation about the relationship between `theme.extend` and `defaultTheme` [here](https://github.com/RyanClementsHax/tailwindcss-themer/blob/main/docs/config.md#this-plugins-config-overwrites-what-is-in-the-normal-tailwind-config-n-collision). In short, you're good to continue to use `theme.extend` all you want until you need it themed by this plugin. Whatever you want themed by this plugin needs to be in the plugin's config. This is just because of a limitation in tailwind's plugin api.
